exercises/06-rock-paper-scissors.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Rock Paper Scissors</title>
+</head>
+<body>
+  <button onclick="
+    userChoice = 'rock';
+    playTheGame();
+    "
+  >Rock</button>
+
+  <button onclick="
+    userChoice = 'paper';
+    playTheGame();
+"
+  >Paper</button>
+
+  <button onclick="
+    userChoice = 'scissors';
+    playTheGame();
+"
+  >Scissors</button>
+
+  <script>
+    const choices = ['rock', 'paper', 'scissors']
```

Comment:
The usual best practice for values that are not changeable in run time is to define it as a constant
```JS
  const CHOICES = [...]
```